constants.js

```diff
@@ -1,5 +1,84 @@
 import { Enum } from "./enum.js";
 
-const Nation = Enum.fromArray(["AH", "IT", "FR", "GB", "GE", "RU"], "Nation");
+export const Nation = Enum.fromArray(
+  ["AH", "IT", "FR", "GB", "GE", "RU"],
+  "Nation"
+);
 
-export { Nation };
+export const NationCard = [
+  [Nation.GE, Nation.IT],
+  [Nation.RU, Nation.FR],
+  [Nation.AH, Nation.GE],
+  [Nation.IT, Nation.GB],
+  [Nation.FR, Nation.AH],
+  [Nation.GB, Nation.RU],
+].reduce((map, [k, v]) => {
+  map.set(k, v);
+  return map;
+}, new Map());
```

Comment:
I'm just curious, might it be preferable/more readable to initialize the new `Map` directly with the arguments rather than doing a `reduce`?
```javascript
export const NationCard = new Map(
  [
    [Nation.GE, Nation.IT],
    [Nation.RU, Nation.FR],
    [Nation.AH, Nation.GE],
    [Nation.IT, Nation.GB],
    [Nation.FR, Nation.AH],
    [Nation.GB, Nation.RU],
  ]
);
```

Reply:
Oop, I didn't know that was possible! This changes everything